ansible/cluster-setup/README.md

```diff
@@ -35,7 +35,13 @@ A guide to installing `Assemble with Ansible`
 1. Login to OpenShift
 1. If you want to use the GitHub integration, generate a Personal Access Token for GitHub and set the `GITHUB_TOKEN` environment variable.
    See the official [Backstage Documentation](https://backstage.io/docs/getting-started/configuration#setting-up-a-github-integration) for information on how to create one.  For the purposes of a demonstration, a Personal Access Token will do.
-1. If you want to use GitHub as an IDP for backstage then create an GitHub app and set the `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` environment variables.
+1. Create a [Github Organization](https://github.com/settings/organizations) and set the `GITHUB_ORGANIZATION` environment variable to the name of the Organization. You may also use any organization you are a member of, as long as you have the ability to create new repositories within it.
+1. In order to create a webhook you'll need to Create a new GitHub Application.  You can do so by clicking [here](https://github.com/organizations/:org/settings/apps/new?url=https://janus-idp.io/blog&webhook_active=false&public=false&administration=write&checks=write&actions=write&code=read&commit_statuses=write&dependabot_alerts=write&dependabot_secrets=write&deployments=write&discussions=write&environments=write&issues=write&packages=write&pages=write&pull_requests=write&reopository_hooks=write&repository_projects=write&secret_scanning_alerts=write&secrets=write&security_events=write&workflows=write&webhooks=write) and replacing the `:org` part of the URL with your organization name before opening.
```

Comment:
This is really confusing in my opinion. This flow expects the users to click the `here` link then look for the `:org` inside the link and replace it. And I feel like most people are just going to click the link get a 404 and assume the README is wrong (I almost posted a comment that your link was broken)

If we want to go this approach I would recommend something like

You can create the webhook by using the following link
`https://github.com/organizations/<ORGANIZATION>/settings/apps/new?url=https://janus-idp.io/blog&webhook_active=false&public=false&administration=write&checks=write&actions=write&code=read&commit_statuses=write&dependabot_alerts=write&dependabot_secrets=write&deployments=write&discussions=write&environments=write&issues=write&packages=write&pages=write&pull_requests=write&reopository_hooks=write&repository_projects=write&secret_scanning_alerts=write&secrets=write&security_events=write&workflows=write&webhooks=write` and replacing `<ORGANIZATION>` with your organizations name.

I know it looks messier but if we expect them to change part of a url I think we need to give them the URL not a link

Reply:
That's fair.  I'll change the verbiage and not make it a hyperlink

Reply:
This might just be, but when I'm a user going through these steps I like to not have to manually edit the URL.  Since we're setting the org as an env var anyways, could we take the same approach we're using in other places and provide the user with something they can throw into the command line to get that full url? Then they can just copy/paste twice and be done with it
Something like: `echo "https://github.com/organizations/$GITHUB_ORGANIZATION/settings/apps/new?url=https://janus-idp.io/blog&webhook_active=false&public=false&administration=write&checks=write&actions=write&code=read&commit_statuses=write&dependabot_alerts=write&dependabot_secrets=write&deployments=write&discussions=write&environments=write&issues=write&packages=write&pages=write&pull_requests=write&reopository_hooks=write&repository_projects=write&secret_scanning_alerts=write&secrets=write&security_events=write&workflows=write&webhooks=write"`